skimage/data/__init__.py

```diff
@@ -592,6 +592,65 @@ def checkerboard():
     return _load("data/chessboard_GRAY.png")
 
 
+def cells3D():
+    """3D microscopy images of cells.
+
+    The returned data is a 3D array with the physical dimensions provided in
+    ``(z, y, x)`` order. Each voxel has a size of ``(0.29 0.26 0.26)``
+    micrometer.
+
+    Returns
+    -------
+    cells3D: (60, 256, 256) float64 ndarray
+        The volumetric images of cells taken with an optical microscope.
+
+    Notes
+    -----
+    The data for this was provided by the Allen Institute for Cell Science.
+
+    It has been downsampled by a factor of 4 in the row and column dimensions
+    to reduce computational time.
+
+    The microscope reports the following voxel spacing
+
+        * Original voxel size is ``(0.290, 0.065, 0.065)``.
+        * Scaling factor is ``(1, 4, 4)`` in each dimension.
+        * After rescaling the voxel size is ``(0.29 0.26 0.26)``.
+    """
+
+    return _load("data/cells.tif")
+
+
+def human_mitosis():
+    """A dataset containing an image of human cells undergoing mitosis.
+
+    Returns
+    -------
+    human_mitosis: (512, 512) uint8 ndimage
+        Data of human cells undergoing mitosis taken from [1]_.
+
+    Notes
+    -----
+
+    Copyright David Root. Licensed under CC-0[2]_.
```

Comment:
```suggestion
    Copyright David Root. Licensed under CC-0 [2]_.
```

Reply:
Regarding all the metadata, maybe we could point to the `AS_09125_050116030001_D03f00d0.tif` entry at https://gitlab.com/scikit-image/data/-/blob/master/README.md#data in a 'single' source perspective?

Reply:
I would rather copy over what is missing. Do you want me to point to anything else that you've included in the README?

Reply:
Ok. I didn't mean that anything was missing (more like the opposite).

Maybe this sentence isn't correct: "Data of human cells undergoing mitosis taken from [1]_." Strictly speaking, the image wasn't taken from that paper, it was taken from CellProfiler as detailed here: https://scikit-image.org/docs/dev/auto_examples/applications/plot_human_mitosis.html#segment-human-cells-in-mitosis
(The image is part of the data used in the cited paper, but does not ship _as is_ with the paper, even considering the supplemental material, etc.)

Reply:
hmm, thanks for the details.

I think the context is very important. I'll try to add that phrase in.

That said, renaming the fiels in registry broke some tests too. so this will take a little longer than i had hoped for today.

Reply:
Maybe they don't need renaming...? :p